browser/src/control/Control.PartsPreview.js

```diff
@@ -88,7 +88,7 @@ L.Control.PartsPreview = L.Control.extend({
 					}, this), 500);
 				}
 
-				var bottomBound = this._getBottomBound();
+				this._getBottomBound();
```

Comment:
Unusual to have a getFoo() function that doesn't return anything; is that expected and/or can we rename it or ? ... needs someone like Gokay to review this not me of course =)

Reply:
I think this one is forgotten while checking the unused variables :) 

@hcvcastro let me approve this PR so you don't lose time.

This will be complete if you can have time for a one line follow up for the forgotten line.

Thanks :)

Reply:
Yes, I am still cleaning more code, I will simplify and rename functions